--- a/Rules    0000-00-00 00:00:00.000000000 -0000
+++ b/Rules    0000-00-00 00:00:00.000000000 -0000
@@ -10,17 +10,30 @@
 #   item, use the pattern "/about/*/"; "/about/*" will also select the parent,
 #   because "*" matches zero or more characters.

-compile '/stylesheet/' do
+compile '/css/*/' do
   # don't filter or layout
 end

+# for NanDoc:
+compile '/README/*' do
+  filter :erb
+  filter :kramdown
+  layout 'default'
+end
+
 compile '*' do
   filter :erb
   layout 'default'
 end

-route '/stylesheet/' do
-  '/style.css'
+# for NanDoc:
+route '/css/*' do
+  if item.children.empty?
+    item.identifier =~ /^(.+)\/$/
+    "#{$1}.css"
+  else
+    fail("don't know what to do with folders in css/: #{item.identifier}")
+  end
 end

 route '*' do
